tests/e2e/pageObjects/CreateSpacePage.js

```diff
@@ -0,0 +1,53 @@
+require("dotenv").config();
+
+const { format } = require("util");
+
+const config = require("../../cucumber.conf.js");
+
+class CreateSpacePage {
+
+  constructor() {
+    this.baseUrl = config.tandoorURL;
+    this.searchURL = this.baseUrl + `search/`;
+    this.spaceOverviewURL = this.baseUrl + `space-overview`;
+    this.createSpaceBtnSelector = `//input[@value="Create Space"]`;
+    this.spaceNameInputSelector = `#id_create-name`;
+    this.succesMsgSelector = `//div[@class="alert alert-success alert-dismissible fade show"]`;
```

Comment:
not everything in the class should be used as selector. so choose the one that best identifies the element
```suggestion
    this.succesMsgSelector = `//div[contains(@class,"alert-success")]`;
```